.github/workflows/releaseNightly.yaml

```diff
@@ -0,0 +1,29 @@
+name: Nightly release
+
+# # Pushes `nightly` image tag as scheduled.
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.release-variables.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configurator version
+        id: release-variables
+        run: |
+          export CONFIG_VERSION=nightly-$(git rev-parse --short HEAD)
+          echo ::set-output name=version::$CONFIG_VERSION
+
+  build-push:
+    name: Build and Push images to Docker Hub
+    needs: version
+    uses: ./.github/workflows/_push_image.yaml
+    with:
+      version: ${{needs.version.outputs.version}} 
+      image-tag: nightly
+      image-overwrite-latest: false
```

Comment:
Here makes more sense the comment I left before: Having two different workflows for pre-releasing and releasing.

Reply:
the _push_image actions it can be used for both. The one is doing the pre-release flow we are used to is the releaseManual one